README.md

```diff
@@ -110,7 +110,7 @@ def mystery6(s)
 end
 ```
 
-- What is mystery6("goodnight moon")?
-- What is mystery6("Ada Developers Academy")?
-- What is mystery6("Hi, there!")?
+- What is mystery6("goodnight moon")? "moon goodnight"
+- What is mystery6("Ada Developers Academy")? "Ada Developers Ada"
+- What is mystery6("Hi, there!")? m6("") + "there! Hi,"
```

Comment:
minor, but notable: all of these answers should have a leading space. " moon goodnight", " Academy Developers Ada", " there! Hi,"